Framework/src/TaskRunner.cxx

```diff
@@ -436,6 +437,7 @@ void TaskRunner::startCycle()
 
 void TaskRunner::finishCycle(DataAllocator& outputs)
 {
+  ILOG(Debug, Ops) << "Finish cycle " << mCycleNumber << " in " << mTaskConfig.taskName << ENDM;
```

Comment:
No need to say where, "Facility" fields says that. In standalone, the device name is at the beginning of the line.

Reply:
correct, fixed.